Sources/ConfigCat/ConfigCatClient.swift

```diff
@@ -294,12 +299,10 @@ public final class ConfigCatClient: NSObject, ConfigCatClientProtocol {
 
             let details = self.evaluate(setting: setting, key: key, user: evalUser, fetchTime: result.fetchTime)
             guard let typedValue = details.value as? Value else {
-                let message = String(format: """
-                                             The value '%@' cannot be converted to the requested type.
-                                             Returning defaultValue: [%@].
-                                             Here are the available keys: %@
-                                             """, "\(details.value)", "\(defaultValue)", [String](result.settings.keys))
-                self.log.error(message: message)
+                let message = String(format: "Failed to evaluate setting '%@' (the value '%@' cannot be converted to the requested type). "
+                    + "Returning the `defaultValue` parameter that you specified in your application: '%@'. Available keys: %@."
+                    key, "\(details.value)", "\(defaultValue)", [String](result.settings.keys))
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`